Libraries/LibWeb/Bindings/CanvasRenderingContext2DWrapper.cpp

```diff
@@ -397,5 +397,16 @@ JS::Value CanvasRenderingContext2DWrapper::put_image_data(JS::Interpreter& inter
     return JS::js_undefined();
 }
 
+JS::Value CanvasRenderingContext2DWrapper::canvas_getter(JS::Interpreter& interpreter)
+{
+    auto* impl = impl_from(interpreter);
+    if (!impl)
+        return {};
+    auto* element = impl->element();
+    if (!element)
+        return JS::js_null();
+    return wrap(interpreter.heap(), static_cast<Element&>(*element));
```

Comment:
Why the cast to `Element` here? It should be fine to pass a `HTMLCanvasElement` to `wrap()`, no? :)

Reply:
hah indeed - I took that from `getElementById`

https://github.com/SerenityOS/serenity/blob/1a1394f7a2f17bb0631cc4aaaa9c78f0cf700312/Libraries/LibWeb/Bindings/DocumentWrapper.cpp#L87